README.md

```diff
@@ -3,13 +3,13 @@
 [![Build Status](https://travis-ci.org/bitcrowd/bitstyles.svg?branch=master)](https://travis-ci.org/bitcrowd/bitstyles)
 [![npm version](https://badge.fury.io/js/bitstyles.svg)](https://badge.fury.io/js/bitstyles)
 
-A collection of tiny Sass CSS objects and helpers, Bitstyles is intended to be used by reasonably CSS- & HTML-aware frontend developers. This is no simple drop-in solution to skin a website, like [Bootstrap](http://getbootstrap.com) or [Foundation](http://foundation.zurb.com), rather a collection of focused object-classes, each defining just a single visual aspect. By using these simple building blocks in your HTML, it’s possible to construct complex layouts without repeating CSS.
+## Documentation
 
-There is now a layer of complex components built on top of the low-level classes, see the documentation below.
+For the full documentation, visit [https://bitcrowd.github.io/bitstyles/](https://bitcrowd.github.io/bitstyles/).
 
-## Documentation
+A collection of tiny Sass CSS objects and helpers, Bitstyles is intended to be used by reasonably CSS- & HTML-aware frontend developers. This is no simple drop-in solution to skin a website, rather a collection of focused object-classes, each defining just a single visual aspect. By using these simple building blocks in your HTML, it’s possible to construct complex layouts without repeating CSS.
```

Comment:
Just a vague suggestion: this is a pretty accurate description of the project/repo. Maybe it should come **before** the `Documentation` headline as the actual description of the project:

```
# Bitstyles

... badges

<description>

## Documentation
```